From 8a9712e2b6e62afe26cd8d0c64940b21c8bbeba6 Mon Sep 17 00:00:00 2001
From: Neta Zur <neta@marvell.com>
Date: Wed, 20 Jul 2016 09:05:06 +0300
Subject: [PATCH 0620/2241] dts: a70x0: remove Armada-7040-RZ board device tree
 files from Makefile

The 7040-rz board is not supported anymore

Change-Id: Iee11dcff5cb00ac1883be1f017d27534d3d014a8
Signed-off-by: Neta Zur <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31260
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/Makefile | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
index dc9df37..894a5a5 100644
--- a/arch/arm64/boot/dts/marvell/Makefile
+++ b/arch/arm64/boot/dts/marvell/Makefile
@@ -7,11 +7,6 @@ dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sgmii1.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sfp.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-db-router.dtb
-# For Armada-7040-Z1 revision
-dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-rz-db-router.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-rz-db-nand.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-rz-db-rxaui.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-rz-pd.dtb
 
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db-router.dtb
 
-- 
2.7.4

